CHANGES.rst

```diff
@@ -200,34 +196,123 @@ API changes
+      This is also a Bugfix. [#4152, #4270]
+
+  - ``NDData``: added a ``meta``-getter, which will set and return an empty
+    OrderedDict if no meta was previously set. [#4509, #4469]
 
-  - ``NDData``: ``Masked_Quantity`` objects are allowed as ``data``
-    parameter. [#4270]
+  - ``NDData``: added an ``meta``-setter. It requires that the meta is
+    dictionary-like (it also accepts Headers or ordered dictionaries and others)
+    or None. It will copy the given value before saving it. [#4509, #4469]
 
-  - ``NDUncertainty``: Added ``array``, ``unit`` and ``copy`` as optional
-    parameters. [#4272]
+  - ``NDArithmeticMixin``: The operand in arithmetic methods (``add``, ...)
+    doesn't need to be a subclass of ``NDData``. It is sufficient if it can be
+    converted to one. This conversion is done internally. [#4272]
 
-  - ``NDUncertainty``: Public methods ``propagate_add``, etc. are replaced by
-    a general ``propagate`` method. [#4272]
+  - ``NDArithmeticMixin``: The arithmetic methods allow several new arguments to
+    control how or if different attributes of the class will be processed during
+    the operation. [#4272]
 
-  - ``NDUncertainty`` and subclasses: implement a representation (``__repr__``). [#4787]
+  - ``NDArithmeticMixin``: Giving the parameter ``propagate_uncertainties`` as
+    positional keyword is deprecated and will be removed in the future. You now
+    need to specify it as keyword-parameter. Besides ``True`` and ``False`` also
+    ``None`` is now a valid value for this parameter. [#4272, #4851]
 
-  - ``StdDevUncertainty``: added an optional parameter ``copy`` which is False
-    by default. [#4272]
+  - ``NDArithmeticMixin``: The wcs attribute of the operands is not compared and
+    thus raises no ValueError if they differ, except if a ``compare_wcs``
+    parameter is specified. [#4272]
 
-  - ``NDArithmeticMixin``: Added ``handle_mask``, ``handle_meta`` and
-    ``compare_wcs`` and ``uncertainty_correlation`` as optional parameters.
-    [#4272]
+  - ``NDArithmeticMixin``: The arithmetic operation was split from a general
+    ``_arithmetic`` method to different specialized private methods to allow
+    subclasses more control on how the attributes are processed without
+    overriding ``_arithmetic``. The ``_arithmetic`` method is now used to call
+    these other methods. [#4272]
 
-  - ``NDArithmeticMixin``: The optional ``propagate_uncertainties`` parameter
-    can also be ``None``. [#4272]
+  - ``NDUncertainty``: ``support_correlated`` attribute is deprecated in favor of
+    ``supports_correlated`` which is a property. Also affects
+    ``StdDevUncertainty``. [#4272]
 
-  - ``NDArithmeticMixin``: Does not compare the wcs for equality except if
-    ``compare_wcs`` is given as function that compares it. [#4272]
+  - ``NDUncertainty``: added the ``__init__`` that was previously implemented in
+    ``StdDevUncertainty`` and takes an additional ``unit`` parameter. [#4272]
```

Comment:
I just realized this may be backwards-incompatible because I inserted the unit-parameter before the existing copy parameter. Should I reorder the parameters so it's backwards-compatible? And if so, do it in this PR or in a seperate one?